.github/workflows/ci.yml

```diff
@@ -238,8 +238,9 @@ jobs:
           tools: none
 
       - name: Install java
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
+          distribution: zulu
```

Comment:
For reference, see https://github.com/actions/setup-java/blob/v2/docs/switching-to-v2.md.